src/metadata-stream.js

```diff
@@ -6,12 +6,24 @@
 (function(window, videojs, undefined) {
   'use strict';
   var
```

Comment:
This is way out of the scope of this review, so no changes requested here, just curious. Is there a reason for this beyond style choice?